data/common.yaml

```diff
@@ -16,12 +16,19 @@ unbound::record: {}
 unbound::access:
   - '::1'
   - '127.0.0.1/8'
+unbound::do_not_query_address:
```

Comment:
Are localhost addresses the correct default here?  We don't currently have anything set, so this might be a surprise for folks who run a authoritative server locally on another port perhaps.